### DISCUSSION TITLE
The application can't be opened - and bunch crashes

### DISCUSSION THREAD

**Discussion by klrichardson** [ttscoff/bunch-docs#116] [Q&A]
I have a very simple bunch (just getting started that I am asking to open 1 app. It opens the app but then bunch hangs and then it crashes with the error "The Application can't be opened -50".

I fear I am doing something wrong. Any help appreciated. This app has SO MUCH potential for me. Thank you for this Brett!

**Comment by ttscoff** [comment]
What app is it?

-Brett

**Reply by klrichardson** [reply]
Cisco AnyConnect Secure Mobility Client

**Reply by theeChucker** [reply]
I had the same issue.  I had a syntax error that was causing it.  Turned out to be an incorrect comment.

**Reply by klrichardson** [reply]
Thanks @95omega and @ttscoff. Simple mistake on my part not giving enough of a pause as a part of the process.

**Reply by klrichardson** [reply]
Spoke too soon. Can't seem to figure out where my issue is. Have shared my bunch below with some personal information changed to see if my syntax is off

```
---
title: Startup Bunch
---
# Bunch for daily startup

@@

# Preferences
(Show Dock)
(Dnd on)

# Start apps
Cisco AnyConnect Secure Mobility Client
- {Enter !15}
- [***********]
- {Enter !15}
- {Enter}
Safari
- safari:https://www.website.com
- [xxxxxx@website.com]
- {tab}
- [**************]
- {Enter !10}
safari:safari:https://www.website2.com
safari:safari:https://www.website3com
safari:safari:https://www.website4.com
Messages
Slack
Drafts
Fantastical
Quip
OmniFocus
ATEM Software Control
TextExpander
```

**Reply by ttscoff** [reply]
Couple general things, though I'm not sure exactly which problem we're trying to solve right now. Is this Bunch still crashing Bunch? If so, can you share Bunch Logs\* or crash logs?

\* **Help->Set Log Level->Debug**, restart Bunch, and use **Help->Open Log** (⇧+⌘+L).

If you're adding file lines (`- xxx`) to a browser, you don't need the browser prefix, so the url directly under `Safari` can just be `- https://website.com`. In the subsequent lines that aren't file lines under Safari, you have the `safari:` prefix twice, which shouldn't be necessary (and I'm not even sure it would work). Maybe a typo in copying the Bunch? 

I can imagine that if you're sending a url to safari first and then typing text such as a login, execution sequence might get really screwy (waiting for a url to load, then expecting the login field to be focused right away). That might be a situation where you want to create an external automation specific to this command, e.g. a Keyboard Maestro or Keysmith macro, and then trigger it from inside the Bunch.

BTW, you can include typed strings inside of curly brackets by double quoting them and can string multiple keys/strings together, e.g. your Cisco keystroke commands could be simplified to one command: `{Enter !15 "*********" Enter !15 Enter}`

**Reply by klrichardson** [reply]
Thanks so much Brett! I'm new to both bunch and scripting but this helps a LOT. I've now setup my morning routine to work for me as a first step with the help of Bunch and KM. Appreciate your work AND your help.
